app/scripts/modules/netflix/availability/availability.trend.component.ts

```diff
@@ -102,7 +115,7 @@ export class AvailabilityTrendController implements ng.IComponentController {
         () => {
           this.popoverOpen[index] = false;
         },
-        300);
+        500);
```

Comment:
Heh, for context the tooltips are small enough that 300 felt like not enough time to get the cursor to the popup.